examples/demo-simple-stf/tests/stf_test.rs

```diff
@@ -26,6 +27,13 @@ impl From<[u8; 32]> for DaAddress {
     }
 }
 
+impl FromStr for DaAddress {
+    type Err = anyhow::Error;
+    fn from_str(_s: &str) -> Result<Self, Self::Err> {
+        unimplemented!()
```

Comment:
Is it ok?

Reply:
oh yeah, since this is just for a test. and we never call that. I can give a proper implementation as well, but the test case doesn't really use it. I can implement it if we think it's necessary (just need to copy any of the u8;32 addresses that we already have)

Reply:
Someone can start using it in the future, so we should provide the implementation.

Reply:
Done!